style.css

```diff
@@ -0,0 +1,39 @@
+label{
+    display: block;
+}
+ body{
```

Comment:
to make the code readable and according to conventions i suggest you to please add a blank line after every block and a space between class/id/tag name and curly braces.